docs/env.yml

```diff
@@ -9,9 +9,9 @@ dependencies:
   - r-data.table=1.14.2
   - r-jsonlite=1.7.2
   - r-knitr=1.37
+  - r-markdown
```

Comment:
I'm planning a follow-up PR updating the version constraints on all the `conda` packages we pull in in this project's CI. I'll put an constraint on `r-markdown` then.

For now, I think it's fine to let this float and have the `conda` solver figure out an appropriate version.